ir/src/symbol_table.rs

```diff
@@ -151,6 +159,23 @@ impl SymbolTable {
         Ok(())
     }
 
+    pub(super) fn insert_random_values(
+        &mut self,
+        values: &RandomValues,
+    ) -> Result<(), SemanticError> {
+        // TODO: add name of random values vector to the identifiers. For now it can be called only
+        // by `$rand`
+        if values.size() > 0 {
```

Comment:
Instead of relying on `values.size()`, I'd probably add an accessor method - something like `has_bindings()`.